ClassLibrary/ClassLibrary/Library.cs

```diff
@@ -41,23 +45,51 @@ public List<Author> Authors
         } 
             = new List<Author>();
 
-        public Author AddAuthor(String nameAuthor,String lastNameAuthor)
+        public void AddAuthorInList(Author author)
         {
-            if (nameAuthor.IsNullOrWhiteSpace() || lastNameAuthor.IsNullOrWhiteSpace())
+            if (author == null)
             {
-                throw new NullReferenceException("The author's first or last name is empty ");
+                throw new NullReferenceException("Author's reference is null");
             }
 
-            Author returnAuthor = Authors?.FirstOrDefault(author =>
-                author.Name == nameAuthor && author.LastName == lastNameAuthor);
+            Author returnAuthor = Authors?.FirstOrDefault(x =>
```

Comment:
`returnAuthor` is not the best name for this variable.

Reply:
What idea do you have?

Reply:
try to think about it yourself, it's also ok to leave it as is.